### PR TITLE
Raises exception when record not found

### LINKED ISSUE #23: TypeCastError when trying to get
Running this

``` crystal
class MenuItem < ActiveRecord::Model
  adapter postgres

  primary(id :: Int)
  field(seq :: Int)
  field(name :: String)
end

MenuItem.get(1)
```

Gives the following TypeCastError

```
cast to MenuItem failed (TypeCastError)
[4356167138] *CallStack::unwind:Array(Pointer(Void)) +82
[4356167041] *CallStack#initialize<CallStack>:Array(Pointer(Void)) +17
[4356167000] *CallStack::new:CallStack +40
[4356205665] *Exception@Exception#initialize<TypeCastError, String>:CallStack +33
[4356205597] *TypeCastError#initialize<TypeCastError, String>:CallStack +29
[4356205532] *TypeCastError::new<String>:TypeCastError +92
[4356243251] *MenuItem@ActiveRecord::Model::get<Int32>:MenuItem +163
[4356139493] __crystal_main +26229
[4356148723] main +35
```

Crystal 0.9.0 (Sun Oct 18 19:49:05 UTC 2015)

### PR DESCRIPTION
**Pull request by sdogruyol**
This fixes #23

### CODE REVIEW

File: src/model.cr [hunk 1]
```diff
@@ -189,7 +191,11 @@ module ActiveRecord
     end
 
     def self.get(primary_key)
-      build(adapter.get(primary_key)) as self
+      unless adapter.get(primary_key) == nil
```
Comment (waterlink) [inline src/model.cr]: You could have used just `if` here without `==` or `!=`

**Comment by waterlink** [comment]
Looks good to me. Left one small suggestion. And looks like it is time for me to use formatter too.

**Comment by sdogruyol** [comment]
Done :+1:

**Comment by waterlink** [comment]
Thanks!

**Pull request merged**